ship-it-inc/ship-it-fend
# #167962480 implement google login page user interface

#### What does this PR do?
It implements google login page to the app
#### Description of Task to be completed?
- implement the ui
- add the login functionalities
#### How should this be manually tested?
- Fetch the branch `ft-implement-login-167962480`
- Check out into the branch `ft-implement-login-167962480`
- run npm run dev
- You should be directed to the login ui page.

#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
[https://www.pivotaltracker.com/story/show/167962480](#167962480)
#### Screenshots (if appropriate)
Nil
